core/src/main/java/org/openstack4j/api/networking/ext/LbaasV2Service.java

```diff
@@ -0,0 +1,28 @@
+package org.openstack4j.api.networking.ext;
+
+/**
+ * LBaaS (Load Balancer as a Service) V2 support
+ * @author emjburns
+ */
+public interface LbaasV2Service {
```

Comment:
Ya, thinking about it, I definitely agree with removing the v2 from all method names. Good call! I'm gonna do a refactor now.